Source/Cake.Coveralls/Cake.Coveralls.csproj

```diff
@@ -19,31 +19,24 @@
       <Description>Cake AddIn that extends Cake with ability to post Code Coverage results to Coveralls.io.</Description>
       <PackageLicenseExpression>MIT</PackageLicenseExpression>
       <PackageProjectUrl>https://github.com/cake-contrib/Cake.Coveralls/</PackageProjectUrl>
-      <PackageTags>cake;script;build;coveralls;cake-addin</PackageTags>
+      <PackageTags>cake;script;build;coveralls;cake-addin;cake-build;addin</PackageTags>
       <RepositoryUrl>https://github.com/cake-contrib/Cake.Coveralls.git</RepositoryUrl>
       <PackageReleaseNotes>https://github.com/cake-contrib/Cake.Coveralls/releases/tag/$(Version)</PackageReleaseNotes>
-  </PropertyGroup>
-  <PropertyGroup>
-    <CodeAnalysisRuleSet>Cake.Coveralls.ruleset</CodeAnalysisRuleSet>
+      <CodeAnalysisRuleSet>..\Cake.Coveralls.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
-    <None Remove="icon.png" />
+    <CakeContribGuidelinesOmitRecommendedConfigFile Include="stylecop.json" />
   </ItemGroup>
-
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="1.0.0" PrivateAssets="All" />
-    <PackageReference Include="Cake.Addin.Analyzer" Version="0.1.3">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
```

Comment:
Is there a reason you removed `Cake.Addin.Analyzer`?

Reply:
The package in question is indeed outdated; the latest version is [0.1.3](https://www.nuget.org/packages/Cake.Addin.Analyzer/0.1.3#versions-body-tab), released [on 03.03.21](https://github.com/WormieCorp/Cake.Addin.Analyzer/releases). I removed it due to its lack of support for .NET 6+, as critical errors were encountered during project solution upgrades.
Consequently, it fails to analyze the Cake infrastructure accurately within the Visual Studio environment.
Additionally, this package is not an official one.

Reply:
Ok. I can follow your reasoning.

I'll create an issue "here" that we re-add it, once it has been updated (I agree, it is also one of those repos that has not have much love lately) and I'll also raise an issue "there" about the fact that is no longer working.